api/src/main/kotlin/bomber/arango/ArangoDao/JsonScenarioDao.kt

```diff
@@ -0,0 +1,117 @@
+package bomber.arango.ArangoDao
+
+import com.arangodb.ArangoCollectionAsync
+import com.arangodb.ArangoDBAsync
+import com.arangodb.ArangoDatabaseAsync
+import com.arangodb.entity.BaseDocument
+import com.arangodb.model.AqlQueryOptions
+import com.arangodb.util.MapBuilder
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+
+/**
+ * This dao is responsible for crud operation scenarious
+ * @author kostya05983
+ */
+class JsonScenarioDao {
+    companion object {
+        private const val DB_NAME = "bomber"
+        private const val COLLECTION_NAME = "jsonRestCollection"
+
+        private const val SELECT_QUERY = "FOR document in jsonRestCollection LIMIT @@offset @@limit return t"
+    }
+
+    private var arangoDb: ArangoDBAsync = ArangoDBAsync.Builder().build()
+    private var db: ArangoDatabaseAsync
+    private var collection: ArangoCollectionAsync
+    private val objectMapper = ObjectMapper()
+
+    init {
+        arangoDb.createDatabase(DB_NAME).get()
+        db = arangoDb.db(DB_NAME)
+        db.createCollection(COLLECTION_NAME).get()
+        collection = db.collection(COLLECTION_NAME)
+    }
+
+    private enum class VarFields(val text: String) {
```

Comment:
how it works?

Reply:
aa, i'm understand, it is simple way for constant declaring=) Am i right?

Reply:
Yes, you are right, It is used for constant.